ui/src/project.vue

```diff
@@ -502,6 +562,39 @@ export default {
                 this.$router.replace("/project/"+this.selected._id+"/"+this.tabs[this.tab].id);
             }
         },
+        detailTab: function() {
+            if(this.detailTab == 4) {
+                this.axios.get("/comment",{params : {
+                    project : this.selected._id
+                }}).then(res=>{
+                    var url = Vue.config.event_ws+"/subscribe?jwt="+Vue.config.jwt;
+                    this.ws = new ReconnectingWebSocket(url, null, {reconnectInterval: 3000});
```

Comment:
I thought project page is already subscribing to project.update.* events. Wouldn't this compete with that? I think you should bind to both events through a single connection instead, and act accordingly based on which event you receive.